test/bricks-net-url.test.cc

```diff
@@ -0,0 +1,261 @@
+/*******************************************************************************
```

Comment:
It should be a symlink but this would require cloning Bricks to the target machine during REPL install. This could be achieved via a postinstall npm script.